json-examples/slice-provisionning.json

```diff
@@ -0,0 +1,137 @@
+{
+  "ietf-network-slice-service:network-slice-services": {
+    "slo-sle-templates": {
+      "slo-sle-template": [
+        {
+          "id": "low-latency-template",
+          "template-description": "lowest possible latencey forwarding behavior"
+        }
+      ]
+    },
+    "slice-service": [
+      {
+        "service-id": "Slice EMBB_UP",
+        "service-description": "Dedicate TN Slice for EMBB-UP",
+        "slo-sle-template": "low-latency-template",
+        "status": {},
+        "sdps": {
+          "sdp": [
+            {
+              "sdp-id": "sdp1",
+              "ac-ref": [
```

Comment:
```suggestion
              "ietf-ac-glue:ac-ref": [
```

Reply:
Can t we have the augmentation native in the same DM (ietf-ac-svc:attachment-circuits) ?

Reply:
We can, but this would means that the base AC will always import the service modules., which I'm trying to avoid.